pkg/authenticator/config/validate_test.go

```diff
@@ -37,15 +37,49 @@ func TestValidate(t *testing.T) {
 			},
 			assert: assertEmptyErrorList(),
 		},
+		{
+			description: "happy path - jwt",
```

Comment:
Added some UT for env variables validation @eladkug

Reply:
why you removed filepath and update host?

Reply:
Because until now there was only k8s authentication so we could put random string in UT like file-path


And now when we have a factory that check if string includes authn-k8s or authn-jwt to decide authentication flow